### ISSUE TITLE
Add support for v0.7.0

### ISSUE THREAD

**Issue by knadh** [bumi/listmonk-heroku#4]
Hi @bumi. Could you update the template + instructions for the latest release, [v0.7.0](https://github.com/knadh/listmonk/releases/tag/v0.7.0-alpha)?

The new setup should be much simpler. A new built-in BasicAuth negates the need for Nginx, and bulk of the config moving to the UI means very few env variables to configure as a well.

**Comment by bumi** [comment]
whoot that's super cool! amazing changelog! 
I will look into it.

**Comment by knadh** [comment]
Hi @bumi. Just wondering if you'd get a chance to update this.

**Comment by bumi** [comment]
Sorry for the delay. will do it asap, sorry.

**Comment by bumi** [comment]
I am getting this error (on master) why I try to deploy to heroku: any idea?

```
2020/10/26 17:26:53 init.go:118: unable to initialize embedded filesystem: no ID found in the file
2020/10/26 17:26:53 init.go:119: using local filesystem for static assets
2020/10/26 17:26:53 init.go:138: failed to initialize local file for assets: stat frontend/dist/favicon.png: no such file or directory
```

**Comment by bumi** [comment]
ah I guess the build task for the frontend was not properly executed.

**Comment by bumi** [comment]
Giving up for now to compile the binary when creating the heroku app. 

But I've updated this repo and the included binary to use the latest 0.7.0-alpha release. It is still running behind nginx but using listmonk for authentication.

**Comment by knadh** [comment]
Thanks. `make dist` builds the frontend, backend, and packs the release ready binary. However, the simplest approach would be to just pull the latest release from the listmonk releases page, extract it, and run the binary, rather than committing the binary in the repo.

I'm not familiar with Heroku, but I'll do some digging and see if I can send a PR for this.

**Comment by bumi** [comment]
That's what I now did. 

but first I tried to make heroku build the binary which would make it easier to update and keep the deploy button in sync with the releasaes.

But I did not manage to make heroku run the correct make commands. 
Heroku uses [buildpacks](https://devcenter.heroku.com/articles/buildpacks) and the go buildpack runs some `go install` commands.

basically all this is is a Heroku [app.json](https://github.com/bumi/listmonk-heroku/blob/master/app.json) that tells heroku how to configure and deploy the app. ([app.json documentaton](https://devcenter.heroku.com/articles/app-json-schema)
This can be part of the main repo, which I tried [here](https://github.com/bumi/listmonk/commit/321a403d6bab138ec99a25f8fab3e63a2e7ae261) (without the experiments to run the make commands)

But anyway this works right now, and maybe we can optimize it in the future and build the binary from source on heroku
